src/plugins/sessions/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+
+/**
+ * A plugin to open documents in the main area.
```

Comment:
Maybe something like this to describe the plugin?

```suggestion
 * A plugin to stop the kernels, sessions and terminals polling
```